### PR TITLE
Add more information on complex queries.

### PR DESCRIPTION
**Pull request by markstory**
Add links to the query builder docs for running complex queries. Sometimes the query builder just gets in the way, and SQL is the simplest solution.

### CODE REVIEW

File: en/orm/query-builder.rst [hunk 1]
```diff
@@ -1181,6 +1181,16 @@ subqueries::
...
 the ``select()`` and ``join()`` methods.
 
+Executing Complex Queries
+-------------------------
+
+While the query builder makes it easy to build most queries, very complex
+queries can be tedious and complicated to build. You may want to :ref:`execute
+the desired SQL directly <running-select-statements>`.
+
+Executing SQL allows you to fine tune the query that will be run, but doesn't
```
Comment (ravage84) [inline en/orm/query-builder.rst]: > Executing SQL...

Something is missing :confused: 
Like the sentense or its meaning is not complete.

Comment (markstory) [inline en/orm/query-builder.rst]: How about now?

Comment (ravage84) [inline en/orm/query-builder.rst]: Ah yes, better. But how about: "Executing SQL directly"? (nitpicking)

Comment (markstory) [inline en/orm/query-builder.rst]: done.

**Comment by antograssiot** [comment]
@cake17 @HavokInspiration french translation added.
@markstory :+1:

**Comment by cake17** [comment]
@antograssiot french translation is correct to me :+1:

**Pull request merged**